lib/blockscore/base.rb

```diff
@@ -60,8 +60,16 @@ def persisted?
       !id.nil? && !deleted?
     end
 
+    def deleted?
```

Comment:
I think we should consider setting up mutant to run as part of the CI process. We could configure it to run on each commit, and test all changes to code made since that commit. This should be all we need to do:

```diff
diff --git a/config/mutant.yml b/config/mutant.yml
index 9a18437..5675a65 100644
--- a/config/mutant.yml
+++ b/config/mutant.yml
@@ -3,3 +3,4 @@ name: blockscore
 namespace: BlockScore
 expect_coverage: 1/1
 ignore_subjects: []
+since: HEAD~1
```